vignettes/admiral.Rmd

```diff
@@ -32,17 +32,23 @@ library(admiral)
 data("dm")
 data("ds")
 data("ex")
+```
 
+```{r, eval=TRUE}
 # derive treatment variables
 adsl <- dm %>%
   mutate(TRT01P = ARMCD, TRT01A = ACTARMCD) %>%
   derive_var_trtsdtm(dataset_ex = ex) %>%
-  mutate(TRTSDT = date(TRTSDTM)) %>%
+  derive_vars_dtm_to_dt(vars(TRTSDTM)) %>%
   derive_var_trtedtm(dataset_ex = ex) %>%
-  mutate(TRTEDT = date(TRTEDTM)) %>%
+  derive_vars_dtm_to_dt(vars(TRTEDTM)) %>%
```

Comment:
You can transform both treatment start and end date in one go: `derive_vars_dtm_to_dt(vars(TRTSDTM, TRTEDTM))`.

Reply:
Done.